src/test/java/com/wikia/webdriver/testcases/Anonymization/AnonymizationTests.java

```diff
@@ -0,0 +1,171 @@
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.global_navitagtion.NavigationBar;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.minieditor.MiniEditorComponentObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.DetachedSignInPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.SignInPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.messagewall.MessageWall;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.ArticleHistoryPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.anonymization.SpecialAnonymizationUserPage;
+
+import org.joda.time.DateTime;
+import org.testng.annotations.Test;
+
+import java.time.LocalDate;
+
+@SuppressWarnings("SpellCheckingInspection")
+@InBrowser(browser = Browser.CHROME)
+@Execute(onWikia = "communitytest")
+@Test(groups = "anonymization")
+public class AnonymizationTests extends NewTestTemplate {
+
+
+  private static final String ERROR_MESSAGE =
+      "We don't recognize these credentials. Try again or register a new account.";
+
+  @Test
+  public void anonymizedUserCannotLogin() {
+
+    Credentials credentials = new Credentials();
+    String timestamp = Long.toString(DateTime.now().getMillis());
+    String qanon = "QAanon" + timestamp;
+    String passw = "makak";
+    SignUpUser
+        user = new SignUpUser(qanon, credentials.emailAnonymousUserTestWikia, passw,
+                              LocalDate.of(1990, 3, 19));
+    UserRegistration.registerUserEmailConfirmed(user);
+
+    SpecialAnonymizationUserPage anonymizationStaff = new SpecialAnonymizationUserPage().open();
+    anonymizationStaff.loginAs(User.STAFF);
+    anonymizationStaff.fillFutureAnon(qanon)
+        .submitAnonymization();
+
+    Assertion.assertStringContains(anonymizationStaff.getAnonConfirmation(), qanon);
+
+    anonymizationStaff.logOut();
+
+    SignInPage signIn = openLoginModalOnDesktop();
+    signIn.login(qanon, passw);
+    Assertion.assertEquals(signIn.getError(), ERROR_MESSAGE);
+  }
+
+  private SignInPage openLoginModalOnDesktop() {
+    return new DetachedSignInPage(new NavigationBar().clickOnSignIn());
+  }
+
+
+  @Test
+  public void anonymizationWikiEdits() {
+
+    Credentials credentials = new Credentials();
+    String timestamp = Long.toString(DateTime.now().getMillis());
+    String qanon = "QAanon" + timestamp;
+    String passw = "makak";
+    SignUpUser
+        user = new SignUpUser(qanon, credentials.emailAnonymousUserTestWikia, passw,
+                              LocalDate.of(1990, 3, 19));
+    UserRegistration.registerUserEmailConfirmed(user);
+
+    new WikiBasePageObject().loginAs(user.getUsername(), user.getPassword(), wikiURL);
+
+    new ArticleContent(user.getUsername()).push("Test"+ timestamp, "AnonymizationTest");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "AnonymizationTest" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)